client/my-sites/plugins/plugin-item/plugin-item.jsx

```diff
@@ -267,14 +266,14 @@ module.exports = React.createClass( {
 				<div className="plugin-item__meta is-placeholder"></div>
 			</CompactCard>
 		);
-	},
+	}
 
-	onItemClick( event ) {
+	onItemClick = ( event ) => {
 		if ( this.props.isSelectable ) {
 			event.preventDefault();
```

Comment:
I'm not sure why the author wanted to [cancel the event from propagating](https://developer.mozilla.org/en/docs/Web/API/Event/preventDefault), but I think we can get rid of this. For example, a popover may not disappear with a click on on this.